### ISSUE TITLE
Appveyor failing on 64-bit architecture

### ISSUE THREAD

**Issue by satwikkansal** [coala/coala-quickstart#166]
Links: https://ci.appveyor.com/project/coala/coala-quickstart/build/1.0.132/job/rttmk9vcdsq9rl3b and https://ci.appveyor.com/project/coala/coala-quickstart/build/1.0.134/job/e7jbpvs4jcxko07c

Part of the logs
```
running build_ext
    building 'lxml.etree' extension
    creating build\temp.win-amd64-3.4
    creating build\temp.win-amd64-3.4\Release
    creating build\temp.win-amd64-3.4\Release\src
    creating build\temp.win-amd64-3.4\Release\src\lxml
    c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\Bin\amd64\cl.exe /c /nologo /Ox /MD /W3 /GS- /DNDEBUG -Isrc\lxml\includes -Ic:\python34-x64\include -Ic:\python34-x64\include /Tcsrc\lxml\lxml.etree.c /Fobuild\temp.win-amd64-3.4\Release\src\lxml\lxml.etree.obj -w
    cl : Command line warning D9025 : overriding '/W3' with '/w'
    lxml.etree.c
    src\lxml\includes\etree_defs.h(14) : fatal error C1083: Cannot open include file: 'libxml/xmlversion.h': No such file or directory
    Compile failed: command 'c:\\Program Files (x86)\\Microsoft Visual Studio 10.0\\VC\\Bin\\amd64\\cl.exe' failed with exit status 2
    creating Users
    creating Users\appveyor
    creating Users\appveyor\AppData
    creating Users\appveyor\AppData\Local
    creating Users\appveyor\AppData\Local\Temp
    creating Users\appveyor\AppData\Local\Temp\1
    c:\Program Files (x86)\Microsoft Visual Studio 10.0\VC\Bin\amd64\cl.exe /c /nologo /Ox /MD /W3 /GS- /DNDEBUG -I/usr/include/libxml2 /TcC:\Users\appveyor\AppData\Local\Temp\1\xmlXPathInitj4yre3yk.c /FoUsers\appveyor\AppData\Local\Temp\1\xmlXPathInitj4yre3yk.obj
    xmlXPathInitj4yre3yk.c
    C:\Users\appveyor\AppData\Local\Temp\1\xmlXPathInitj4yre3yk.c(1) : fatal error C1083: Cannot open include file: 'libxml/xpath.h': No such file or directory
    *********************************************************************************
    Could not find function xmlCheckVersion in library libxml2. Is libxml2 installed?
    *********************************************************************************
    error: command 'c:\\Program Files (x86)\\Microsoft Visual Studio 10.0\\VC\\Bin\\amd64\\cl.exe' failed with exit status 2
    
    ----------------------------------------
Command "c:\python34-x64\python.exe -u -c "import setuptools, tokenize;__file__='C:\\Users\\appveyor\\AppData\\Local\\Temp\\1\\pip-build-2gvlnfuy\\lxml\\setup.py';f=getattr(tokenize, 'open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'))" install --record C:\Users\appveyor\AppData\Local\Temp\1\pip-8yi1aygi-record\install-record.txt --single-version-externally-managed --compile" failed with error code 1 in C:\Users\appveyor\AppData\Local\Temp\1\pip-build-2gvlnfuy\lxml\
Command exited with code 1
```

importance/high

**Comment by satwikkansal** [comment]
Upstream issue filed at https://github.com/vfaronov/httpolice/issues/5

**Comment by satwikkansal** [comment]
Temporary workaround for this is to pin the version of lxml to `3.6.0`  in our `requirements.txt`. This build https://ci.appveyor.com/project/satwikkansal/coala-quickstart/build/job/c4sf9lthnecqepp3  does the same and is successful.

**Comment by satwikkansal** [comment]
There exists https://bugs.launchpad.net/lxml/+bug/1704707 for the missing wheel file, #167 solves this temporarily by pinning lxml version to `3.6.0`. Once https://bugs.launchpad.net/lxml/+bug/1704707 is resolved, changes in #167 can be reverted back!
